src/gwkokab/inference/jenks.py

```
@@ -0,0 +1,318 @@
+    float
+        The loss percentage for the bucket, calculated as described above.
+    """
+
+    total = sum(bucket)
+    return (len(bucket) * max(bucket) - total) / (len(bucket) * max(bucket)) * 100.0
+
+
+def _total_loss(subsets: Sequence[Sequence[int]]) -> float:
+    total_loss = (
+        (
+            sum([len(subset) * max(subset) for subset in subsets])
```

Pylint (3.3.7)

[refactor] Consider using a generator instead 'sum(len(subset) * max(subset) for subset in subsets)'

(R1728)